src/database/queries/tag.js

```diff
@@ -0,0 +1,21 @@
+import knex from '../knex'
+import * as _ from './utilities'
```

Comment:
`_` is a weird variable name. The libraries `underscore.js` and `lowdash.js` are being fun and cleaver but you shouldn't :P 

If the file is called `utilities` I would call this variable `utilities`. But you could also just import the functions you need. 

```js
import { createRecord, deleteAll, findAll, findAllWhere, deleteRecord } from './utilities'
```